### PR TITLE
feat: usb video class (webcam) extension

### PR DESCRIPTION
**Pull request by USA-RedDragon**
Fixes: #295
Related PR: https://github.com/siderolabs/pkgs/pull/865

### CODE REVIEW

File: drivers/v4l-uvc/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: v1alpha1
+metadata:
+  name: v4l-uvc
+  version: "$VERSION"
+  author: Jacob McSwain
```
Comment (USA-RedDragon) [inline drivers/v4l-uvc/manifest.yaml:5]: Should this be me or a code owner?

File: drivers/v4l-uvc/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Verifiying
+
+You can verify the modules are enabled by reading the `/proc/modules` where it _should_ show the module is live.
+
+For example:
+
+```
+❯ talosctl -n 192.168.42.15  read /proc/modules
+TODO
```
Comment (USA-RedDragon) [inline drivers/v4l-uvc/README.md:26]: TODO

File: drivers/v4l-uvc/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+In addition, you should be able to verify presence of the video device if the USB device is plugged in, checking `/dev` directory.
+
+For example:
+
+```
+❯ talosctl -n 192.168.42.15 ls /dev | grep video
+TODO
```
Comment (USA-RedDragon) [inline drivers/v4l-uvc/README.md:35]: TODO

File: drivers/v4l-uvc/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+You can also verify everything in dmesg:
+
+```
+❯ talosctl -n 192.168.42.15 dmesg
+# look for lines like these:
+TODO
```
Comment (USA-RedDragon) [inline drivers/v4l-uvc/README.md:43]: TODO

File: drivers/v4l-uvc/README.md (hunk 2 above)
Comment (USA-RedDragon) [reply drivers/v4l-uvc/README.md]: Resolved in https://github.com/siderolabs/extensions/pull/296/commits/5ac513c2387c7ba4089f80a42b89218759fad0ce

File: drivers/v4l-uvc/README.md (hunk 3 above)
Comment (USA-RedDragon) [reply drivers/v4l-uvc/README.md]: Resolved in https://github.com/siderolabs/extensions/pull/296/commits/5ac513c2387c7ba4089f80a42b89218759fad0ce

File: drivers/v4l-uvc/README.md (hunk 4 above)
Comment (USA-RedDragon) [reply drivers/v4l-uvc/README.md]: Resolved in https://github.com/siderolabs/extensions/pull/296/commits/5ac513c2387c7ba4089f80a42b89218759fad0ce

**Review by frezbo — APPROVED** [review]

**Pull request merged**